opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp

```diff
@@ -55,11 +55,11 @@ namespace Opm
         size_t numWells(size_t timestep) const;
         size_t getMaxNumCompletionsForWells(size_t timestep) const;
         bool hasWell(const std::string& wellName) const;
-        WellPtr getWell(const std::string& wellName) const;
-        std::vector<WellPtr> getOpenWells(size_t timeStep) const;
+        WellPtr getWell(const std::string& wellName);
+        std::vector<WellPtr> getOpenWells(size_t timeStep);
         std::vector<WellConstPtr> getWells() const;
         std::vector<WellConstPtr> getWells(size_t timeStep) const;
-        std::vector<WellPtr> getWells(const std::string& wellNamePattern) const;
+        std::vector<WellPtr> getWells(const std::string& wellNamePattern);
```

Comment:
> the wells and schedule stuff is unrelated to the tables refactoring, right?

Yes - but the container `OrderedMap<T>` was victim of some '*try to use references - dammit*' refactorings.